frontend/src/layout/navigation/Breadcrumbs/breadcrumbsLogic.tsx

```diff
@@ -1,40 +1,20 @@
 import { kea } from 'kea'
-import { organizationLogic } from '../../../scenes/organizationLogic'
-import { teamLogic } from '../../../scenes/teamLogic'
+import { organizationLogic } from 'scenes/organizationLogic'
+import { teamLogic } from 'scenes/teamLogic'
 import './Breadcrumbs.scss'
 import { breadcrumbsLogicType } from './breadcrumbsLogicType'
-import { sceneLogic } from '../../../scenes/sceneLogic'
-import { Scene } from '../../../scenes/sceneTypes'
-import { urls } from '../../../scenes/urls'
-import { preflightLogic } from '../../../scenes/PreflightCheck/logic'
-import { identifierToHuman, stripHTTP } from '../../../lib/utils'
-import { userLogic } from '../../../scenes/userLogic'
+import { sceneLogic } from 'scenes/sceneLogic'
+import { preflightLogic } from 'scenes/PreflightCheck/logic'
+import { identifierToHuman, stripHTTP } from 'lib/utils'
+import { userLogic } from 'scenes/userLogic'
 import React from 'react'
-import { Lettermark } from '../../../lib/components/Lettermark/Lettermark'
-import { ProfilePicture } from '../../../lib/components/ProfilePicture'
-import { dashboardsModel } from '../../../models/dashboardsModel'
-import { featureFlagLogic } from '../../../scenes/feature-flags/featureFlagLogic'
-import { personsLogic } from '../../../scenes/persons/personsLogic'
-import { asDisplay } from '../../../scenes/persons/PersonHeader'
-import { PopupProps } from 'lib/components/Popup/Popup'
+import { Lettermark } from 'lib/components/Lettermark/Lettermark'
+import { ProfilePicture } from 'lib/components/ProfilePicture'
 import { ProjectSwitcherOverlay } from '~/layout/navigation/ProjectSwitcher'
 import { OrganizationSwitcherOverlay } from '~/layout/navigation/OrganizationSwitcher'
-import { sceneConfigurations } from 'scenes/scenes'
+import { Breadcrumb } from '~/types'
 
-export interface Breadcrumb {
-    /** Name to display. */
-    name: string | null | undefined
-    /** Symbol, e.g. a lettermark or a profile picture. */
-    symbol?: React.ReactNode
-    /** Path to link to. */
-    path?: string
-    /** Whether this breadcrumb refers to the current location. */
-    here?: boolean
-    /** Whether to show a custom popup */
-    popup?: Pick<PopupProps, 'overlay' | 'sameWidth' | 'actionable'>
-}
```

Comment:
Why not keep the `Breadcrumb` type here? `types.ts` is a monstrously large collection of anything  while `breadcrumbsLogic.tsx` is the actual heart of breadcrumbs, so if there are no problems with importing this file (and there's always `import type`), I'd advocate for avoiding `types.ts` bloat

Reply:
It's much easier to import from `types.ts`, than a specific deeply nested logic. I tried moving it back just now, but the amount of custom imports, when you're importing already 10 other types from `types.ts`, felt very hacky. Plus if the type is in a different file than the logic, typegen imports it directly instead of requiring it to be passed via `<>`. So I'll keep as is.

Reply:
I see, fine with `types.ts` for now then, although it is getting bigger and bigger and bigger